ui-showdetails/src/main/java/app/tivi/showdetails/details/ShowDetails.kt

```diff
@@ -367,7 +359,7 @@ private fun ShowDetailsScrollingContent(
             }
 
             seasons.forEach {
-                SeasonWithEpisodesRow(
+                seasonWithEpisodesRow(
```

Comment:
I get that this *isn't* a composable function but the naming feels wrong. Might be worth a wider discussion on this?